### ISSUE TITLE
Readme examples possibly incorrect

### ISSUE THREAD

**Issue by SevereOverfl0w** [logaan/vlad#20]
It appears that you've changed the syntax slightly, and not remembered to update the readme!

e.g. 
```clojure
(def validation
  (attr [:age] present))
```
Should really be
```clojure
(def validation
  (attr [:age] (present)))
```

Or is this a regression of some kind?

**Comment by SevereOverfl0w** [comment]
Another error I just noticed:
https://github.com/logaan/vlad#translation `assign-name` is actually `assign-names`

**Comment by logaan** [comment]
Thanks for noticing! I'll fix these examples up shortly.

**Comment by logaan** [comment]
Resolved in d2e36c1
